invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html

```diff
@@ -0,0 +1,257 @@
+    </p>
+    <ul>
+      <li>Aktuellste</li>
+      <li>Beste Übereinstimmung</li>
+    </ul>
+    <h3>Reguläre Ausdrücke</h3>
+    <p>
+      Reguläre Ausdrücke sind eine mächtige Sprache für den Mustervergleich, mit der
+      man nach bestimmten Mustern in einem Feld suchen kann. Wenn wir zum Beispiel
+      alle Datensätze mit dem DOI-Präfix 10.5281 finden wollen, können wir eine Suche mit
+      regulären Ausdrücken verwenden:
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=metadata.subjects.identifier:%2F03yrm5c2%5B1%2C6%5D%2F"
+        ><code>metadata.subjects.identifier:/03yrm5c2[1,6]/</code></a
+      >
+    </p>
+    <p>
+      Vorsicht, der reguläre Ausdruck muss mit dem <em>gesamten</em> Feldwert übereinstimmen.
+      Weitere Einzelheiten finden Sie in der
+      <a
+        href="https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html#regexp-syntax"
+        >Syntax des regulären Ausdrucks</a
+      >
+      .
+    </p>
+    <h3>Fehlende Werte</h3>
+    <p>
+      Es ist möglich, mit den Feldnamen <code>_exists_</code> und
+      <code>_missing_</code> nach Datensätzen zu suchen,
+      in denen entweder ein Wert fehlt oder die einen Wert in einem bestimmten Feld haben.
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=_missing_:metadata.additional_titles"
+        ><code>_missing_:metadata.additional_titles</code></a
+      >
+      (alle Datensätze ohne metadata.additional_titles)
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=_exists_:metadata.creators"
+        ><code>_exists_:metadata.creators</code></a
+      >
+      (alle Datensätze mit metadata.creators)
+    </p>
+    <h3>Erweiterte Konzepte</h3>
+    <h4>Boosten</h4>
+    <p>
+      Sie können den Boost-Operator <code>^</code> verwenden, wenn ein Begriff relevanter ist als ein
+      anderer. Sie können z. B. nach allen Datensätzen suchen, die den Ausdruck
+      <em>open science</em> entweder <em>im title</em> oder im
+      <em>description</em> feld enthalten, aber Datensätze mit dem Ausdruck im
+      <em>title</em> feld höher einstufen:
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a
+        href="/search?page=1&amp;size=20&amp;q=metadata.title:%22open%20science%22%5E5%20metadata.description:%22open%20science%22"
+        ><code>metadata.title:"open science"^5 metadata.description:"open science"</code></a
+      >
+    </p>
+    <h4>Unschärfe</h4>
+    <p>
+      Mit dem Fuzzy-Operator <code>~</code>können Sie nach
+      Begriffen suchen, die Ihrem Suchbegriff ähnlich, aber nicht genau gleich sind.
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=oepn~"><code>oepn~</code></a>
+    </p>
+    <p>
+      Die Ergebnisse entsprechen Datensätzen mit ähnlichen Begriffen wie <code>oepn</code> die z. B.
```

Comment:
I'm not sure whether the typo is confusing without explaining that it is a 'wanted' typo (oepn).